src/js/components/ChatFoot/ChatFoot.js

```diff
@@ -35,24 +36,28 @@ const ChatFoot = (props) => {
 };
 
 const ChatInput = (props) => {
-  const { title } = props;
-  const maxInputHeight = 132;
+  const { id, title } = props;
+  const { state, setChatInputState } = useContext(GlobalContext);
   const refInput = useRef(null);
-  const [inputState, setInputState] = useState({
-    chatInput: '',
-    currentInputHeight: 22,
-  });
-  const { chatInput, currentInputHeight } = inputState;
+
+  const maxInputHeight = 132;
+  let chatInput = '';
+  let currentInputHeight = 22;
+
+  if (state[id]) {
```

Comment:
Could you destructure this above?

Reply:
Nah, I cant destructure it because the key changes. If I try, I get errors and Eslint doesnt like it.